scwconfig/load.go

```diff
@@ -34,13 +36,15 @@ func Load() (Config, error) {
 		}
 		confV1, err := unmarshalConfV1(content)
 		if err == nil {
+			logger.Warningf("loaded config V1 from %s: config V1 is deprecated, please switch your config file to the V2", configPath)
 			return confV1.toV2().catchInvalidProfile()
 		}
 		confV2, err := unmarshalConfV2(content)
```

Comment:
You don't need to catch the `error` of `unmarshalConfV1` before unmarshal the v2 config? It could lead to a v2 error message for a v1 config.

Reply:
This error happens when file does not exist. At this point in the code we want to try for V2 before returning an error.